mix.exs

```diff
@@ -31,7 +31,7 @@ defmodule EctoEnum.Mixfile do
   end
 
   defp deps do
-    [{:ecto, "~> 2.1.0"},
+    [{:ecto, "~> 2.2"},
      {:postgrex, "~> 0.13.0", optional: true},
      {:mariaex, "~> 0.8.0", optional: true},
```

Comment:
I think

```elixir
{:postgrex, "~> 0.13.0", optional: true},
{:mariaex, "~> 0.8.0", optional: true},
```

should be 

```elixir
```elixir
{:postgrex, "~> 0.13", optional: true},
{:mariaex, "~> 0.8", optional: true},
```